content/en/docs/tasks/debug-application-cluster/monitor-node-health.md

```diff
@@ -151,31 +135,28 @@ with new rule definition:
 }
 ```
 
-### Change Log Path
-
-Kernel log in different OS distros may locate in different path. The `log`
-field in `config/kernel-monitor.json` is the log path inside the container.
-You can always configure it to match your OS distro.
-
-### Support Other Log Format
+### Change log path
```

Comment:
```suggestion
### Configure the path for the kernel log device {#kernel-log-device-path}
```

(I think it's OK to break existing hyperlinks because there won't be many of these, and the old title could be misleading).

Reply:
Good idea. I'd like to shorten the heading.